### ISSUE TITLE
External Variant Evidence Sources Ingestion and Display

### ISSUE THREAD

**Issue by wrightmw** [ClinGen/gene-and-variant-curation-tools#62]
We have an existing ticket specifically for bringing in CIViC data for display in the VCI (via the LDH): see #1769 . The purpose of this overarching ticket will be to construct a general workflow that allows for evidence from various different external resources to be automatically displayed in the VCI. CIViC is just one exemplar dataset, but when scoping and specifying this project we need several other exemplar data sets so that we can make sure this project will support multiple different types of data and have maximal utility.

Scoping for this feature provided by Deb:
Scoping for this feature includes:
Display Location: Linked Data Hub (LDH) content could be displayed as a row similar to ClinVar in the Basic Information tab.
Row Content: The row would summarize the content of the Linked Data Hub for that variant. It would provide either a link to the LDH for further exploration, or a direct link to the content LDH obtains (CIViC in this case).

With CIViC data as an example, and as a first step to integration, a user could know from looking at the LDH row in the Basic Information tab, if a variant under curation is (1) present in CIViC and (2) display the evidence available (Evidence IDs, evidence statements or Assertions) in CIViC. The format or specific items to display may change based on the dataset, but the row would be designed to hold some common elements from LDH datasets and minimize development of specific items for additional datasets.

Need to find out from Sai: For datasets LDH would contain, what are some of the common elements? Name of Dataset? Allele Registry ID (Are all variants in the LDH linked by Allele Registry ID?) Other elements?

**Comment by wrightmw** [comment]
Replaced https://github.com/ClinGen/clincoded/issues/2177

**Comment by cgpreston** [comment]
[SP ticket: https://broadinstitute.atlassian.net/browse/CGSP-115](https://broadinstitute.atlassian.net/browse/CGSP-115)

**Comment by cgpreston** [comment]
From Sharon on 7/11/22:  - Also could consider bringing in AllofUs data separately rather than via gnomAD: 

Note that in the reply to the ESP there was a comment about the AllofUs data that will eventually make it into gnomAD but we also talked about bringing it in like we did the PAGE data years ago. Here is the quote - “The ClinGen Variant Curation Interface will then ingest these updated data from gnomAD and directly incorporate these allele counts into the VCI. This will bring awareness of the existence of phenotype data in AoU for any variant with allele counts, allowing curators to access this case-level data for curation with appropriate permissions. This is similar to our prior ingest of the earlier PAGE genotype frequency data into the VCI.”
